README.md

```diff
@@ -40,4 +40,7 @@ Now, you can start using MMT SDK.
 * [Close Position](https://developers.mmt.finance/features/close-position)
 
 For a full detailed technical integration doc, please visit 
-[MMT Developer Docs](https://developers.mmt.finance/clmm-sdk/integrations).
\ No newline at end of file
+[MMT Developer Docs](https://developers.mmt.finance/clmm-sdk/integrations).
+
+### Examples
+please visit here [example](examples/README.md)
```

Comment:
Please update the wording here.

Reply:
A set of examples are provided to walk through the basic usage of the SDK. Please check detailed guideline [here](examples/README.md).